docs/dev/dod.md

```diff
@@ -0,0 +1,16 @@
+# Definition of Done
+These criteria must be met before a feature or artifact can be considered "done".
+
+## Code
+- Code coverage: >75%
+- Codacy grade A
```

Comment:
Its also possible to customize the criterias, which Codacy will check. By default some Problems are allowed to get grade A if the project is big enough.